client/ayon_houdini/startup/python_panels/ayon_generic_loader.pypanel

```diff
@@ -0,0 +1,86 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<pythonPanelDocument>
+  <!-- This file contains definitions of Python interfaces and the
+ interfaces menu.  It should not be hand-edited when it is being
+ used by the application.  Note, that two definitions of the
+ same interface or of the interfaces menu are not allowed
+ in a single file. -->
+  <interface name="ayon_panel" label="AYON Generic Load Panel" icon="MISC_python" showNetworkNavigationBar="false" help_url="">
+    <script><![CDATA[
+import hou
+from PySide2 import QtCore, QtWidgets
```

Comment:
just curious: can we use qtpy instead of PySide2? I am asking this for more innovative development as I remember we used to have done similar import with PySide2 in Substance Painter but the API uses PySide6 after its update.
But it's okay if there is no way to do that.

Reply:
Yes - should be `qtpy`. Future versions of Houdini will also ship PySide6.